recipes/nghttp2/all/conanfile.py

```diff
@@ -7,7 +7,7 @@
 
 
 class Nghttp2Conan(ConanFile):
-    name = "nghttp2"
+    name = "libnghttp2"
```

Comment:
@jgsogo What was the motivation for this? The project calls itself nghttp2?

Reply:
I can't remember right now 😓 Several options come to my mind: Windows or something related to `libcurl` consuming this recipe (https://github.com/jgsogo/conan-center-index/tree/recipe/libcurl/recipes/libcurl). Tomorrow I will have a look at `libcurl` and hope I will realize about it.